example/spec/loadTest/queueGranuleQueuedLoadTest.js

```diff
@@ -0,0 +1,242 @@
+const fs = require('fs');
```

Comment:
What is the intended test target for this test?  We should discuss and consider removing this for this PR unless it meets the specific objective of 2926.

Reply:
Removed queueGranuleQueuedLoadTest